tests/test_tasks.py

```diff
@@ -132,3 +132,37 @@ def test_documents_and_requests(task_name: str, task_class: Task):
             # TODO: Mock lm after refactoring evaluator.py to not be a mess
             for req in requests:
                 assert isinstance(req, Request)
+
+
+def test_arg_string_task_creation():
+    arg_string = "text_target_separator=\n\n,example_separator=\t,save_examples=False"
+    task = tasks.get_task_list_from_args_string(
+        "wnli",
+        template_names=["confident"],
+        task_args=arg_string,
+    )[0]
+    assert task.example_separator == "\t"
```

Comment:
Can we include an assert for `text_target_separator` and also assert on a specific example to check that the string is generated as we expect?

Reply:
Good call. Will do!